examples/vae/main.py

```diff
@@ -127,10 +124,10 @@ def loss_fn(model):
 
 
 @jax.jit
-def eval(model, eval_ds, z):
+def eval(model, eval_ds, z, z_rng):
   xs = eval_ds['image'] / 255.0
   xs = xs.reshape(-1, 784)
-  recon_xs, mean, logvar = model(xs)
+  recon_xs, mean, logvar = model(xs, z_rng)
 
   comparison = jnp.concatenate([xs[:8].reshape(-1, 28, 28, 1),
```

Comment:
Return all samples+reconstructions and subsample+concat outside of eval before saving the image?

Reply:
There are a lot of reconstructions (10,000) so I think pre-processing the data for the comparison image is not a bad idea

Reply:
I didn't realise this. I thought it was going to be just the regular batch size.